### PR TITLE
Book List 

### PR DESCRIPTION
**Pull request by joacomiguens**
## Summary

- Create book-list component.

- Create _variables.scss.

- Create a mixin for card-shadow.

## Notes

I created a new .scss file for variables because I thought that it would help to define, for example, values of "vw" or "vh" like 1280px. 

In my opinion, this change could improve the way of declare and use viewport sizes in many components.

## Screenshot

image

## Trello Card

You can find it [here](https://trello.com/c/86jjuehP/30-listado-de-libros)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,27 +8,87 @@
...
-        <div class="detail__info__item">
-          <h4 class="title">Editorial:</h4>
-          <p class="content">Nombre de la editorial</p>
-        </div>
-        <div class="detail__info__item">
-          <h4 class="title">Año de publicación:</h4>
-          <p class="content">Año de publicación</p>
+      <div class="book-list">
+        <div class="book m-right-bottom-40">
+          <img src="./assets/book-cover.png" alt="" class="book__cover" />
```
Comment (ebedoyaalzate) [inline index.html]: Add alt text

File: scss/_book-list.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  display: flex;
+  flex-direction: column;
+  max-width: 20%;
+  padding: 32px 32px 20px 32px;
+
+  &-list {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 40px auto;
```
Comment (ebedoyaalzate) [inline scss/_book-list.scss]: If you use flexbox don´t need to position with margin auto, instead of use align-item

File: scss/_styles.scss [hunk 3]
```diff
@@ -1,6 +1,25 @@
...
+}
+
+%title {
+  font-size: 16px;
+  font-weight: 700;
+}
+@mixin card-shadow {
+  background-color: $white;
+  box-shadow: 5px 5px 7px 1px $soft-black;
+}
```
Comment (ebedoyaalzate) [inline scss/_styles.scss]: Move this to a file of mixins, and import the file.

File: scss/_styles.scss [hunk 4]
```diff
@@ -1,6 +1,25 @@
...
 
+%content {
+  color: $dark-grey;
+  font-size: 16px;
+}
+
+%title {
+  font-size: 16px;
+  font-weight: 700;
+}
```
Comment (ebedoyaalzate) [inline scss/_styles.scss]: Make a global class file and move this.

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

File: scss/_styles.scss (hunk 4 above)
Comment (joacomiguens) [reply scss/_styles.scss]: Done 😄

File: scss/_styles.scss (hunk 3 above)
Comment (joacomiguens) [reply scss/_styles.scss]: Done 😄

File: scss/_book-list.scss (hunk 2 above)
Comment (joacomiguens) [reply scss/_book-list.scss]: Done 😄

File: index.html (hunk 1 above)
Comment (joacomiguens) [reply index.html]: Done 😄

**Review by ebedoyaalzate — APPROVED** [review]

File: index.html [hunk 5]
```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" type="text/css" href="application.css" />
   </head>
   <body>
+    <div class="container">
     <div class="detail">
```
Comment (mcallegari10) [inline index.html]: Check the indentation for this line 😄

File: scss/_book-list.scss [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+      line-height: 2;
+    }
+
+    &__author {
+      @extend %content;
+    }
+  }
+}
+
+.m-right-bottom-40 {
```
Comment (mcallegari10) [inline scss/_book-list.scss]: I think there's something cleaner for this kind of issue. Check `nth-child` 😉

File: scss/_styles.scss [hunk 7]
```diff
@@ -1,6 +1,14 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;700&display=swap');
 @import 'colors';
-
+@import 'mixins';
```
Comment (mcallegari10) [inline scss/_styles.scss]: This import should go in the `applications.scss`. And the Google Font and color file too

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: scss/_styles.scss (hunk 7 above)
Comment (joacomiguens) [reply scss/_styles.scss]: Done 😄

File: scss/_book-list.scss (hunk 6 above)
Comment (joacomiguens) [reply scss/_book-list.scss]: Done with css > selector 😄

File: index.html (hunk 5 above)
Comment (joacomiguens) [reply index.html]: Done 😄

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**